components/tidb_query/src/rpn_expr/impl_json.rs

```diff
@@ -187,6 +187,38 @@ fn json_extract(args: &[ScalarValueRef]) -> Result<Option<Json>> {
     Ok(j.extract(&path_expr_list))
 }
 
+// Args should be like `(&Option<Json> , &[&Option<Bytes>])`.
+fn json_with_path_validator(expr: &tipb::Expr) -> Result<()> {
```

Comment:
Can you merge the implementation with `json_with_paths_validator`?

Reply:
Done